### PR TITLE
task filter form

### PR DESCRIPTION
**Pull request by goose-life**
image

### CODE REVIEW

**Comment by goose-life** [comment]
Actually I think I'll add other changes to this PR so please don't review it yet :grimacing:

**Comment by goose-life** [comment]
image

**Comment by goose-life** [comment]
Ready for review!

File: indigo_app/forms.py [hunk 1]
```diff
@@ -248,14 +248,19 @@ class TaskFilterForm(forms.Form):
...
     submitted_by = forms.ModelMultipleChoiceField(queryset=User.objects)
     type = forms.MultipleChoiceField(choices=Task.CODES)
+    country = forms.ModelMultipleChoiceField(queryset=Country.objects)
 
     def __init__(self, country, *args, **kwargs):
         self.country = country
         super(TaskFilterForm, self).__init__(*args, **kwargs)
         self.fields['assigned_to'].queryset = User.objects.filter(editor__permitted_countries=self.country).order_by('first_name', 'last_name').all()
         self.fields['submitted_by'].queryset = self.fields['assigned_to'].queryset
+        self.fields['country'].queryset = Country.objects.all()
```
Comment (longhotsummer) [inline indigo_app/forms.py:258]: Line 251 above should obviate this need?

**Review by longhotsummer — APPROVED** [review]

**Comment by longhotsummer** [comment]
Nice

File: indigo_app/forms.py (hunk 1 above)
Comment (goose-life) [inline indigo_app/forms.py:258]:
```suggestion
```

Comment (goose-life) [reply indigo_app/forms.py]: quite right, thank you! I've nuked this

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**